includes/reports_config.js

```diff
@@ -0,0 +1,1385 @@
+    },
+    speedIndex: {
+      name: 'Speed Index',
+      type: 'seconds',
+      singular: 'second',
+      description: 'How quickly the contents of a page are visibly populated. This metric comes from [WebPageTest](https://github.com/WPO-Foundation/webpagetest-docs/blob/main/src/metrics/SpeedIndex.md) and differs slightly from the [Lighthouse version](https://web.dev/speed-index/) (which was based on this).'
+    },
+    bytesCss: {
+      name: 'CSS Bytes',
+      type: 'KB',
+      description: 'The sum of transfer size kilobytes of all external stylesheets requested by the page. An external stylesheets is identified as a resource with the `css` file extension or a MIME type containing `css`.',
+      wpt: {
+        path: 'median$firstView$breakdown$css$bytes',
+        scale: 0.0009765625
+      }
+    },
+    bytesFont: {
+      name: 'Font Bytes',
+      type: 'KB',
+      description: 'The sum of transfer size kilobytes of all fonts requested by the page. A font is identified as a resource with the `eot`, `ttf`, `woff`, `woff2`, or `otf` file extensions or a MIME type containing `font`.',
+      wpt: {
+        path: 'median$firstView$breakdown$font$bytes',
+        scale: 0.0009765625
+      }
+    },
+    bytesHtml: {
+      name: 'HTML Bytes',
+      type: 'KB',
+      description: 'The sum of transfer size kilobytes of all HTML documents requested by the page. An HTML document is identified as a resource with the `html` or `html` file extensions or a MIME type containing `html`.',
+      wpt: {
+        path: 'median$firstView$breakdown$html$bytes',
+        scale: 0.0009765625
+      }
+    },
+    bytesImg: {
+      name: 'Image Bytes',
+      type: 'KB',
+      description: 'The sum of transfer size kilobytes of all external images requested by the page. An external image is identified as a resource with the `png`, `gif`, `jpg`, `jpeg`, `webp`, `ico`, or `svg` file extensions or a MIME type containing `image`.',
+      wpt: {
+        path: 'median$firstView$breakdown$image$bytes',
+        scale: 0.0009765625
+      }
+    },
+    bytesJs: {
+      name: 'JavaScript Bytes',
+      type: 'KB',
+      description: 'The sum of transfer size kilobytes of all external scripts requested by the page. An external script is identified as a resource with the `js` or `json` file extensions or a MIME type containing `script` or `json`.',
+      wpt: {
+        path: 'median$firstView$breakdown$js$bytes',
+        scale: 0.0009765625
+      }
+    },
+    bytesOther: {
+      name: 'Other Bytes',
+      type: 'KB',
+      description: 'The sum of transfer size kilobytes of all unidentified resources requested by the page. An unidentified resource is one that does not match a known script, font, image, stylesheet, XML file, video, HTML document, or plaintext file.',
+      wpt: {
+        path: 'median$firstView$breakdown$other$bytes',
+        scale: 0.0009765625
+      }
+    },
+    bytesTotal: {
+      name: 'Total Kilobytes',
```

Comment:
I found [reports config file](https://github.com/HTTPArchive/httparchive.org/blob/main/config/reports.json) - seems a good idea to keep all the configs in one place (more transparent for future contributors).

I copied it over here (to experiment with) and added the queries.
I wouldn't be able to add the queries unless the format supports multiline strings - so just saved in JS.
Actually it is required to be readable with python - YAML?